<launch>
    <node name="detect_charuco_camera60" pkg="trifinger_cameras" type="detect_charuco_board">
        <remap from="image" to="/camera60/image_raw" />
    </node>

    <node name="detect_charuco_camera180" pkg="trifinger_cameras" type="detect_charuco_board">
        <remap from="image" to="/camera180/image_raw" />
    </node>

    <node name="detect_charuco_camera300" pkg="trifinger_cameras" type="detect_charuco_board">
        <remap from="image" to="/camera300/image_raw" />
    </node>
</launch>
